### PR TITLE
Update run-detection-test.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the result of the detection test. I do not see anywhere to check if it is marked as completed. Ideally If successful, a new alert will appear in a few minutes.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7547

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/run-detection-test.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Run the following PowerShell script on a newly onboarded device to verify that i
    powershell.exe -NoExit -ExecutionPolicy Bypass -WindowStyle Hidden $ErrorActionPreference = 'silentlycontinue';(New-Object System.Net.WebClient).DownloadFile('http://127.0.0.1/1.exe', 'C:\\test-MDATP-test\\invoice.exe');Start-Process 'C:\\test-MDATP-test\\invoice.exe'
    ```
 
-The Command Prompt window will close automatically. If successful, the detection test will be marked as completed and a new alert will appear in the portal for the onboarded device in about 10 minutes.
+The Command Prompt window will close automatically. If successful, a new alert will appear in the portal for the onboarded device in about 10 minutes.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/run-detection-test.md]:
```suggestion
The Command Prompt window will close automatically. If successful, a new alert will appear in the portal for the onboarded device in about ten minutes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**